lib/repository.py

```diff
@@ -17,6 +17,7 @@
 import os
 import urlparse
 import utils
+import shutil
```

Comment:
I've listed this pull as a dependency of https://github.com/open-power-host-os/versions/pull/171.  Good point!